library/src/main/java/com/pengrad/telegrambot/impl/FileApi.java

```diff
@@ -16,6 +16,9 @@
     public FileApi(String token) {
         this.apiUrl = FILE_API + token + "/";
     }
+    public FileApi(String apiUrl, String token) {
```

Comment:
```suggestion

    public FileApi(String apiUrl, String token) {
```
Let's add empty line here